<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveProjectDialog</class>
 <widget class="QDialog" name="SaveProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Project name (prj.name):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="nameLabel_2">
       <property name="text">
        <string>Projects path (dir.projects):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="projectsDirEdit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectProjectsDirButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="simpleCheckBox">
     <property name="text">
      <string>Use simple project saving style (prj.simple, recommended)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="filesCheckBox">
     <property name="text">
      <string>Save the target binary inside the project directory (prj.files)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="gitCheckBox">
     <property name="text">
      <string>Project is a git repo and saving is committing (prj.git)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="zipCheckBox">
     <property name="text">
      <string>Use ZIP format for project files (prj.zip)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
